client/src/api/v2/camera.js

```diff
@@ -165,3 +165,40 @@ export function useCameraRecordingStatus(device) {
 export function getPrettyDeviceName(device) {
   return device === 'primary' ? 'Primary' : 'Secondary';
 }
+
+/**
+ * Initiate receiving camera video feed statuses
+ */
+function initVideoFeedStatus() {
+  emit('send-last-received-camera-video-feed-payloads');
```

Comment:
This is quite a verbose topic. Would `send-video-feed-payloads` be sufficiently descriptive?

Reply:
Done
00a0b9b